oci/private/image_index.sh.tpl

```diff
@@ -37,7 +37,7 @@ function create_oci_layout() {
 
     echo '{"imageLayoutVersion": "1.0.0"}' > "${path}/oci-layout" 
     echo '{"schemaVersion": 2, "manifests": []}' > "${path}/index.json"
-    echo '{"schemaVersion": 2, "mediaType": "application/vnd.oci.image.index.v1+json", "manifests": []}' > "${path}/manifest_list.json"
+    echo '{"schemaVersion": 2, "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json", "manifests": []}' > "${path}/manifest_list.json"
```

Comment:
```suggestion
    echo '{"schemaVersion": 2, "mediaType": "application/vnd.docker.distribution.manifest.v2+json", "manifests": []}' > "${path}/manifest_list.json"
```

When I `docker load` a `oci_tarball` and then inspect the manifest with `docker manifest inspect` I get..

```json
{
        "schemaVersion": 2,
        "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
        "config": {
                "mediaType": "application/vnd.docker.container.image.v1+json",
                "size": 755,
                "digest": "sha256:535d4d587465fe961197cf661e1687d24eb6e2c418ff7b5822ce24b7685d9a2e"
        },
        "layers": [
                {
                        "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                        "size": 803814,
                        "digest": "sha256:e8614d09b7bebabd9d8a450f44e88a8807c98a438a2ddd63146865286b132d1b"
                },
                {
                        "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                        "size": 7975415,
                        "digest": "sha256:c6f4d1a13b699c8490910fd4fd6c7056b90fd0da3077e4f29b4bd27bf0bae6cd"
                },
                {
                        "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                        "size": 9075798,
                        "digest": "sha256:b3843d8722bb9d69a51512b856afdc5e3d407cda23d801e423056e5e0d85cfc6"
                }
        ]
}
```

Reply:
oci_tarball doesn't support output of oci_image_index. you are loading image into docker daemon not image_index.